Broken after update to 4.19 kernel

```
-------------------------------------------------------------------

Installing driver version: 4.4

DisplayLink Linux Software 4.4.24 install script called: install
Distribution discovered: Debian GNU/Linux buster/sid
Installing
Configuring EVDI DKMS module
Registering EVDI kernel module with DKMS
Building EVDI kernel module with DKMS
ERROR (code 3): Failed to build evdi/4.4.24. Consult /var/lib/dkms/evdi/4.4.24/build/make.log for details..

-------------------------------------------------------------------
```
```
todds@todd-debian:~/git/displaylink-debian$ sudo cat /var/lib/dkms/evdi/4.4.24/build/make.log
DKMS make.log for evdi-4.4.24 for kernel 4.19.0-1-amd64 (x86_64)
Fri Dec 28 16:23:16 EST 2018
make KBUILD_VERBOSE=1 SUBDIRS=/var/lib/dkms/evdi/4.4.24/build SRCROOT=/var/lib/dkms/evdi/4.4.24/build CONFIG_MODULE_SIG= -C /lib/modules/4.19.0-1-amd64/build modules
make[1]: Entering directory '/usr/src/linux-headers-4.19.0-1-amd64'
make -C /usr/src/linux-headers-4.19.0-1-amd64 KBUILD_SRC=/usr/src/linux-headers-4.19.0-1-common \
-f /usr/src/linux-headers-4.19.0-1-common/Makefile modules
test -e include/generated/autoconf.h -a -e include/config/auto.conf || (	\
echo >&2;							\
echo >&2 "  ERROR: Kernel configuration is invalid.";		\
echo >&2 "         include/generated/autoconf.h or include/config/auto.conf are missing.";\
echo >&2 "         Run 'make oldconfig && make prepare' on kernel src to fix it.";	\
echo >&2 ;							\
/bin/false)
mkdir -p /var/lib/dkms/evdi/4.4.24/build/.tmp_versions ; rm -f /var/lib/dkms/evdi/4.4.24/build/.tmp_versions/*
make -f /usr/src/linux-headers-4.19.0-1-common/scripts/Makefile.build obj=/var/lib/dkms/evdi/4.4.24/build
(cat /dev/null;   echo kernel//var/lib/dkms/evdi/4.4.24/build/evdi.ko;) > /var/lib/dkms/evdi/4.4.24/build/modules.order
   gcc-8 -Wp,-MD,/var/lib/dkms/evdi/4.4.24/build/.evdi_drv.o.d  -nostdinc -isystem /usr/lib/gcc/x86_64-linux-gnu/8/include -I/usr/src/linux-headers-4.19.0-1-common/arch/x86/include -I./arch/x86/include/generated  -I/usr/src/linux-headers-4.19.0-1-common/include -I./include -I/usr/src/linux-headers-4.19.0-1-common/arch/x86/include/uapi -I./arch/x86/include/generated/uapi -I/usr/src/linux-headers-4.19.0-1-common/include/uapi -I./include/generated/uapi -include /usr/src/linux-headers-4.19.0-1-common/include/linux/kconfig.h -include /usr/src/linux-headers-4.19.0-1-common/include/linux/compiler_types.h  -I/var/lib/dkms/evdi/4.4.24/build -I/var/lib/dkms/evdi/4.4.24/build -D__KERNEL__ -Wall -Wundef -Wstrict-prototypes -Wno-trigraphs -fno-strict-aliasing -fno-common -fshort-wchar -Werror-implicit-function-declaration -Wno-format-security -std=gnu89 -fno-PIE -DCC_HAVE_ASM_GOTO -mno-sse -mno-mmx -mno-sse2 -mno-3dnow -mno-avx -m64 -falign-jumps=1 -falign-loops=1 -mno-80387 -mno-fp-ret-in-387 -mpreferred-stack-boundary=3 -mskip-rax-setup -mtune=generic -mno-red-zone -mcmodel=kernel -funit-at-a-time -DCONFIG_X86_X32_ABI -DCONFIG_AS_CFI=1 -DCONFIG_AS_CFI_SIGNAL_FRAME=1 -DCONFIG_AS_CFI_SECTIONS=1 -DCONFIG_AS_FXSAVEQ=1 -DCONFIG_AS_SSSE3=1 -DCONFIG_AS_CRC32=1 -DCONFIG_AS_AVX=1 -DCONFIG_AS_AVX2=1 -DCONFIG_AS_AVX512=1 -DCONFIG_AS_SHA1_NI=1 -DCONFIG_AS_SHA256_NI=1 -pipe -Wno-sign-compare -fno-asynchronous-unwind-tables -mindirect-branch=thunk-extern -mindirect-branch-register -fno-delete-null-pointer-checks -Wno-frame-address -Wno-format-truncation -Wno-format-overflow -Wno-int-in-bool-context -O2 --param=allow-store-data-races=0 -Wframe-larger-than=2048 -fstack-protector-strong -Wno-unused-but-set-variable -Wno-unused-const-variable -fno-var-tracking-assignments -g -pg -mrecord-mcount -mfentry -DCC_USING_FENTRY -Wdeclaration-after-statement -Wno-pointer-sign -Wno-stringop-truncation -fno-strict-overflow -fno-merge-all-constants -fmerge-constants -fno-stack-check -fconserve-stack -Werror=implicit-int -Werror=strict-prototypes -Werror=date-time -Werror=incompatible-pointer-types -Werror=designated-init -fmacro-prefix-map=/usr/src/linux-headers-4.19.0-1-common/= -Wno-packed-not-aligned  -I/usr/src/linux-headers-4.19.0-1-common/include/drm  -DMODULE  -DKBUILD_BASENAME='"evdi_drv"' -DKBUILD_MODNAME='"evdi"' -c -o /var/lib/dkms/evdi/4.4.24/build/.tmp_evdi_drv.o /var/lib/dkms/evdi/4.4.24/build/evdi_drv.c
   gcc-8 -Wp,-MD,/var/lib/dkms/evdi/4.4.24/build/.evdi_modeset.o.d  -nostdinc -isystem /usr/lib/gcc/x86_64-linux-gnu/8/include -I/usr/src/linux-headers-4.19.0-1-common/arch/x86/include -I./arch/x86/include/generated  -I/usr/src/linux-headers-4.19.0-1-common/include -I./include -I/usr/src/linux-headers-4.19.0-1-common/arch/x86/include/uapi -I./arch/x86/include/generated/uapi -I/usr/src/linux-headers-4.19.0-1-common/include/uapi -I./include/generated/uapi -include /usr/src/linux-headers-4.19.0-1-common/include/linux/kconfig.h -include /usr/src/linux-headers-4.19.0-1-common/include/linux/compiler_types.h  -I/var/lib/dkms/evdi/4.4.24/build -I/var/lib/dkms/evdi/4.4.24/build -D__KERNEL__ -Wall -Wundef -Wstrict-prototypes -Wno-trigraphs -fno-strict-aliasing -fno-common -fshort-wchar -Werror-implicit-function-declaration -Wno-format-security -std=gnu89 -fno-PIE -DCC_HAVE_ASM_GOTO -mno-sse -mno-mmx -mno-sse2 -mno-3dnow -mno-avx -m64 -falign-jumps=1 -falign-loops=1 -mno-80387 -mno-fp-ret-in-387 -mpreferred-stack-boundary=3 -mskip-rax-setup -mtune=generic -mno-red-zone -mcmodel=kernel -funit-at-a-time -DCONFIG_X86_X32_ABI -DCONFIG_AS_CFI=1 -DCONFIG_AS_CFI_SIGNAL_FRAME=1 -DCONFIG_AS_CFI_SECTIONS=1 -DCONFIG_AS_FXSAVEQ=1 -DCONFIG_AS_SSSE3=1 -DCONFIG_AS_CRC32=1 -DCONFIG_AS_AVX=1 -DCONFIG_AS_AVX2=1 -DCONFIG_AS_AVX512=1 -DCONFIG_AS_SHA1_NI=1 -DCONFIG_AS_SHA256_NI=1 -pipe -Wno-sign-compare -fno-asynchronous-unwind-tables -mindirect-branch=thunk-extern -mindirect-branch-register -fno-delete-null-pointer-checks -Wno-frame-address -Wno-format-truncation -Wno-format-overflow -Wno-int-in-bool-context -O2 --param=allow-store-data-races=0 -Wframe-larger-than=2048 -fstack-protector-strong -Wno-unused-but-set-variable -Wno-unused-const-variable -fno-var-tracking-assignments -g -pg -mrecord-mcount -mfentry -DCC_USING_FENTRY -Wdeclaration-after-statement -Wno-pointer-sign -Wno-stringop-truncation -fno-strict-overflow -fno-merge-all-constants -fmerge-constants -fno-stack-check -fconserve-stack -Werror=implicit-int -Werror=strict-prototypes -Werror=date-time -Werror=incompatible-pointer-types -Werror=designated-init -fmacro-prefix-map=/usr/src/linux-headers-4.19.0-1-common/= -Wno-packed-not-aligned  -I/usr/src/linux-headers-4.19.0-1-common/include/drm  -DMODULE  -DKBUILD_BASENAME='"evdi_modeset"' -DKBUILD_MODNAME='"evdi"' -c -o /var/lib/dkms/evdi/4.4.24/build/.tmp_evdi_modeset.o /var/lib/dkms/evdi/4.4.24/build/evdi_modeset.c
   gcc-8 -Wp,-MD,/var/lib/dkms/evdi/4.4.24/build/.evdi_connector.o.d  -nostdinc -isystem /usr/lib/gcc/x86_64-linux-gnu/8/include -I/usr/src/linux-headers-4.19.0-1-common/arch/x86/include -I./arch/x86/include/generated  -I/usr/src/linux-headers-4.19.0-1-common/include -I./include -I/usr/src/linux-headers-4.19.0-1-common/arch/x86/include/uapi -I./arch/x86/include/generated/uapi -I/usr/src/linux-headers-4.19.0-1-common/include/uapi -I./include/generated/uapi -include /usr/src/linux-headers-4.19.0-1-common/include/linux/kconfig.h -include /usr/src/linux-headers-4.19.0-1-common/include/linux/compiler_types.h  -I/var/lib/dkms/evdi/4.4.24/build -I/var/lib/dkms/evdi/4.4.24/build -D__KERNEL__ -Wall -Wundef -Wstrict-prototypes -Wno-trigraphs -fno-strict-aliasing -fno-common -fshort-wchar -Werror-implicit-function-declaration -Wno-format-security -std=gnu89 -fno-PIE -DCC_HAVE_ASM_GOTO -mno-sse -mno-mmx -mno-sse2 -mno-3dnow -mno-avx -m64 -falign-jumps=1 -falign-loops=1 -mno-80387 -mno-fp-ret-in-387 -mpreferred-stack-boundary=3 -mskip-rax-setup -mtune=generic -mno-red-zone -mcmodel=kernel -funit-at-a-time -DCONFIG_X86_X32_ABI -DCONFIG_AS_CFI=1 -DCONFIG_AS_CFI_SIGNAL_FRAME=1 -DCONFIG_AS_CFI_SECTIONS=1 -DCONFIG_AS_FXSAVEQ=1 -DCONFIG_AS_SSSE3=1 -DCONFIG_AS_CRC32=1 -DCONFIG_AS_AVX=1 -DCONFIG_AS_AVX2=1 -DCONFIG_AS_AVX512=1 -DCONFIG_AS_SHA1_NI=1 -DCONFIG_AS_SHA256_NI=1 -pipe -Wno-sign-compare -fno-asynchronous-unwind-tables -mindirect-branch=thunk-extern -mindirect-branch-register -fno-delete-null-pointer-checks -Wno-frame-address -Wno-format-truncation -Wno-format-overflow -Wno-int-in-bool-context -O2 --param=allow-store-data-races=0 -Wframe-larger-than=2048 -fstack-protector-strong -Wno-unused-but-set-variable -Wno-unused-const-variable -fno-var-tracking-assignments -g -pg -mrecord-mcount -mfentry -DCC_USING_FENTRY -Wdeclaration-after-statement -Wno-pointer-sign -Wno-stringop-truncation -fno-strict-overflow -fno-merge-all-constants -fmerge-constants -fno-stack-check -fconserve-stack -Werror=implicit-int -Werror=strict-prototypes -Werror=date-time -Werror=incompatible-pointer-types -Werror=designated-init -fmacro-prefix-map=/usr/src/linux-headers-4.19.0-1-common/= -Wno-packed-not-aligned  -I/usr/src/linux-headers-4.19.0-1-common/include/drm  -DMODULE  -DKBUILD_BASENAME='"evdi_connector"' -DKBUILD_MODNAME='"evdi"' -c -o /var/lib/dkms/evdi/4.4.24/build/.tmp_evdi_connector.o /var/lib/dkms/evdi/4.4.24/build/evdi_connector.c
   gcc-8 -Wp,-MD,/var/lib/dkms/evdi/4.4.24/build/.evdi_encoder.o.d  -nostdinc -isystem /usr/lib/gcc/x86_64-linux-gnu/8/include -I/usr/src/linux-headers-4.19.0-1-common/arch/x86/include -I./arch/x86/include/generated  -I/usr/src/linux-headers-4.19.0-1-common/include -I./include -I/usr/src/linux-headers-4.19.0-1-common/arch/x86/include/uapi -I./arch/x86/include/generated/uapi -I/usr/src/linux-headers-4.19.0-1-common/include/uapi -I./include/generated/uapi -include /usr/src/linux-headers-4.19.0-1-common/include/linux/kconfig.h -include /usr/src/linux-headers-4.19.0-1-common/include/linux/compiler_types.h  -I/var/lib/dkms/evdi/4.4.24/build -I/var/lib/dkms/evdi/4.4.24/build -D__KERNEL__ -Wall -Wundef -Wstrict-prototypes -Wno-trigraphs -fno-strict-aliasing -fno-common -fshort-wchar -Werror-implicit-function-declaration -Wno-format-security -std=gnu89 -fno-PIE -DCC_HAVE_ASM_GOTO -mno-sse -mno-mmx -mno-sse2 -mno-3dnow -mno-avx -m64 -falign-jumps=1 -falign-loops=1 -mno-80387 -mno-fp-ret-in-387 -mpreferred-stack-boundary=3 -mskip-rax-setup -mtune=generic -mno-red-zone -mcmodel=kernel -funit-at-a-time -DCONFIG_X86_X32_ABI -DCONFIG_AS_CFI=1 -DCONFIG_AS_CFI_SIGNAL_FRAME=1 -DCONFIG_AS_CFI_SECTIONS=1 -DCONFIG_AS_FXSAVEQ=1 -DCONFIG_AS_SSSE3=1 -DCONFIG_AS_CRC32=1 -DCONFIG_AS_AVX=1 -DCONFIG_AS_AVX2=1 -DCONFIG_AS_AVX512=1 -DCONFIG_AS_SHA1_NI=1 -DCONFIG_AS_SHA256_NI=1 -pipe -Wno-sign-compare -fno-asynchronous-unwind-tables -mindirect-branch=thunk-extern -mindirect-branch-register -fno-delete-null-pointer-checks -Wno-frame-address -Wno-format-truncation -Wno-format-overflow -Wno-int-in-bool-context -O2 --param=allow-store-data-races=0 -Wframe-larger-than=2048 -fstack-protector-strong -Wno-unused-but-set-variable -Wno-unused-const-variable -fno-var-tracking-assignments -g -pg -mrecord-mcount -mfentry -DCC_USING_FENTRY -Wdeclaration-after-statement -Wno-pointer-sign -Wno-stringop-truncation -fno-strict-overflow -fno-merge-all-constants -fmerge-constants -fno-stack-check -fconserve-stack -Werror=implicit-int -Werror=strict-prototypes -Werror=date-time -Werror=incompatible-pointer-types -Werror=designated-init -fmacro-prefix-map=/usr/src/linux-headers-4.19.0-1-common/= -Wno-packed-not-aligned  -I/usr/src/linux-headers-4.19.0-1-common/include/drm  -DMODULE  -DKBUILD_BASENAME='"evdi_encoder"' -DKBUILD_MODNAME='"evdi"' -c -o /var/lib/dkms/evdi/4.4.24/build/.tmp_evdi_encoder.o /var/lib/dkms/evdi/4.4.24/build/evdi_encoder.c
/var/lib/dkms/evdi/4.4.24/build/evdi_connector.c: In function ‘evdi_get_modes’:
/var/lib/dkms/evdi/4.4.24/build/evdi_connector.c:35:3: error: implicit declaration of function ‘drm_mode_connector_update_edid_property’; did you mean ‘drm_connector_update_edid_property’? [-Werror=implicit-function-declaration]
   drm_mode_connector_update_edid_property(connector, NULL);
   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   drm_connector_update_edid_property
/var/lib/dkms/evdi/4.4.24/build/evdi_connector.c: In function ‘evdi_connector_init’:
/var/lib/dkms/evdi/4.4.24/build/evdi_connector.c:147:2: error: implicit declaration of function ‘drm_mode_connector_attach_encoder’; did you mean ‘drm_connector_attach_encoder’? [-Werror=implicit-function-declaration]
  drm_mode_connector_attach_encoder(connector, encoder);
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  drm_connector_attach_encoder
   ./tools/objtool/objtool orc generate  --module --no-fp --retpoline "/var/lib/dkms/evdi/4.4.24/build/.tmp_evdi_encoder.o";
cc1: some warnings being treated as errors
   ./tools/objtool/objtool orc generate  --module --no-fp --retpoline "/var/lib/dkms/evdi/4.4.24/build/.tmp_evdi_drv.o";
   gcc-8 -Wp,-MD,/var/lib/dkms/evdi/4.4.24/build/.evdi_main.o.d  -nostdinc -isystem /usr/lib/gcc/x86_64-linux-gnu/8/include -I/usr/src/linux-headers-4.19.0-1-common/arch/x86/include -I./arch/x86/include/generated  -I/usr/src/linux-headers-4.19.0-1-common/include -I./include -I/usr/src/linux-headers-4.19.0-1-common/arch/x86/include/uapi -I./arch/x86/include/generated/uapi -I/usr/src/linux-headers-4.19.0-1-common/include/uapi -I./include/generated/uapi -include /usr/src/linux-headers-4.19.0-1-common/include/linux/kconfig.h -include /usr/src/linux-headers-4.19.0-1-common/include/linux/compiler_types.h  -I/var/lib/dkms/evdi/4.4.24/build -I/var/lib/dkms/evdi/4.4.24/build -D__KERNEL__ -Wall -Wundef -Wstrict-prototypes -Wno-trigraphs -fno-strict-aliasing -fno-common -fshort-wchar -Werror-implicit-function-declaration -Wno-format-security -std=gnu89 -fno-PIE -DCC_HAVE_ASM_GOTO -mno-sse -mno-mmx -mno-sse2 -mno-3dnow -mno-avx -m64 -falign-jumps=1 -falign-loops=1 -mno-80387 -mno-fp-ret-in-387 -mpreferred-stack-boundary=3 -mskip-rax-setup -mtune=generic -mno-red-zone -mcmodel=kernel -funit-at-a-time -DCONFIG_X86_X32_ABI -DCONFIG_AS_CFI=1 -DCONFIG_AS_CFI_SIGNAL_FRAME=1 -DCONFIG_AS_CFI_SECTIONS=1 -DCONFIG_AS_FXSAVEQ=1 -DCONFIG_AS_SSSE3=1 -DCONFIG_AS_CRC32=1 -DCONFIG_AS_AVX=1 -DCONFIG_AS_AVX2=1 -DCONFIG_AS_AVX512=1 -DCONFIG_AS_SHA1_NI=1 -DCONFIG_AS_SHA256_NI=1 -pipe -Wno-sign-compare -fno-asynchronous-unwind-tables -mindirect-branch=thunk-extern -mindirect-branch-register -fno-delete-null-pointer-checks -Wno-frame-address -Wno-format-truncation -Wno-format-overflow -Wno-int-in-bool-context -O2 --param=allow-store-data-races=0 -Wframe-larger-than=2048 -fstack-protector-strong -Wno-unused-but-set-variable -Wno-unused-const-variable -fno-var-tracking-assignments -g -pg -mrecord-mcount -mfentry -DCC_USING_FENTRY -Wdeclaration-after-statement -Wno-pointer-sign -Wno-stringop-truncation -fno-strict-overflow -fno-merge-all-constants -fmerge-constants -fno-stack-check -fconserve-stack -Werror=implicit-int -Werror=strict-prototypes -Werror=date-time -Werror=incompatible-pointer-types -Werror=designated-init -fmacro-prefix-map=/usr/src/linux-headers-4.19.0-1-common/= -Wno-packed-not-aligned  -I/usr/src/linux-headers-4.19.0-1-common/include/drm  -DMODULE  -DKBUILD_BASENAME='"evdi_main"' -DKBUILD_MODNAME='"evdi"' -c -o /var/lib/dkms/evdi/4.4.24/build/.tmp_evdi_main.o /var/lib/dkms/evdi/4.4.24/build/evdi_main.c
make[4]: *** [/usr/src/linux-headers-4.19.0-1-common/scripts/Makefile.build:308: /var/lib/dkms/evdi/4.4.24/build/evdi_connector.o] Error 1
make[4]: *** Waiting for unfinished jobs....
   ./tools/objtool/objtool orc generate  --module --no-fp --retpoline "/var/lib/dkms/evdi/4.4.24/build/.tmp_evdi_modeset.o";
   ./tools/objtool/objtool orc generate  --module --no-fp --retpoline "/var/lib/dkms/evdi/4.4.24/build/.tmp_evdi_main.o";
make[3]: *** [/usr/src/linux-headers-4.19.0-1-common/Makefile:1532: _module_/var/lib/dkms/evdi/4.4.24/build] Error 2
make[2]: *** [Makefile:146: sub-make] Error 2
make[1]: *** [Makefile:8: all] Error 2
make[1]: Leaving directory '/usr/src/linux-headers-4.19.0-1-amd64'
make: *** [Makefile:22: all] Error 2
```
This isn't an issue with the script but rather the driver source lagging changes to the 4.19 kernel API.  I'm hoping that this can be fixed by patching the files in [this commit.](https://github.com/DisplayLink/evdi/commit/c7e80bc32c2ef96464613fbbda15fd930bf6fb41) before building evdi.

--- a/displaylink-debian.sh
+++ b/displaylink-debian.sh
@@ -581,7 +581,7 @@ then
 fi
 
 # evdi module still in use (issue 178, 192)
-evdi_version="$(systemctl status dlm.service | grep -o '4.4.[[:digit:]]*')"
+evdi_version="$(dkms status evdi|grep -o '4.4.[[:digit:]]*')"
 dkms remove evdi/$evdi_version --all
 evdi_dir="/usr/src/evdi-$evdi_version"
 if [ -d "$evdi_dir" ];

--- a/displaylink-debian.sh
+++ b/displaylink-debian.sh
@@ -339,16 +339,13 @@ evdi_src_ver="$(echo evdi-* | cut -d'-' -f2)"
 evdi_src="evdi-$evdi_src_ver-src"
 mkdir $evdi_src
 
-# extract $evdi_src.tar.gz to $evdi_src
-tar -xzvf $evdi_src.tar.gz -C $evdi_src --strip 1
-
-# get latest (patched) evdi_connector.c file
-wget https://github.com/DisplayLink/evdi/raw/devel/module/evdi_connector.c -O $evdi_src/evdi_connector.c
-# get latest (patched) evdi_gem.c file
-wget https://raw.githubusercontent.com/DisplayLink/evdi/devel/module/evdi_gem.c -O $evdi_src/evdi_gem.c
-
-# compress $evdi_src.tar.gz from $evdi_src
-tar -zcvf $evdi_src.tar.gz -C $evdi_src .
+evdi_patch_version=v1.5.1
+# get version v1.5.1 of evdi. Note: the version could also be "devel" or "master"
+wget https://github.com/DisplayLink/evdi/archive/$evdi_patch_version.tar.gz -O evdi-$evdi_patch_version.tar.gz
+# extract evdi to $evdi_src
+tar -xzvf evdi-$evdi_patch_version.tar.gz -C $evdi_src --strip 1
+# compress new $evdi_src.tar.gz from $evdi_src/module - this replaces evdi from displaylink-driver package
+tar -zcvf $evdi_src.tar.gz -C $evdi_src/module .
 }
 
 function ver2int {

--- a/displaylink-debian.sh
+++ b/displaylink-debian.sh
@@ -348,7 +348,7 @@ wget https://github.com/DisplayLink/evdi/raw/devel/module/evdi_connector.c -O $e
 wget https://raw.githubusercontent.com/DisplayLink/evdi/devel/module/evdi_gem.c -O $evdi_src/evdi_gem.c
 
 # compress $evdi_src.tar.gz from $evdi_src
-tar -zcvf $evdi_src.tar.gz $evdi_src
+tar -zcvf $evdi_src.tar.gz -C $evdi_src .
 }
 
 function ver2int {

